legate/core/runtime.py

```diff
@@ -1217,19 +1245,39 @@ def find_region_manager(self, region: Region) -> RegionManager:
         assert region in self.region_managers_by_region
         return self.region_managers_by_region[region]
 
-    def destroy_region_manager(
+    def revive_manager(self, region_mgr: RegionManager) -> None:
+        lru_managers: Deque[RegionManager] = deque()
+        for to_check in self.lru_managers:
+            if to_check is not region_mgr:
+                lru_managers.append(to_check)
+        assert len(lru_managers) < len(self.lru_managers)
+        self.lru_managers = lru_managers
```

Comment:
Out of curiosity, did this not work?
```
self.lru_managers.remove(region_mgr)
```

Reply:
that would have worked. (the original code this was copied from needed a loop, but then it has become unnecessary after the code went through some simplification.) will make a follow-on PR to clean this up after the release.